samza-core/src/test/java/org/apache/samza/task/TestAsyncRunLoop.java

```diff
@@ -200,6 +201,45 @@ public void setup() {
     when(consumerMultiplexer.pollIntervalMs()).thenReturn(10);
   }
 
+  @Test
```

Comment:
Would be nice if we also add a test case to illustrate that callback-completed was not incremented by mal-implemented task code.